example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -849,6 +850,54 @@ describe('The S3 Ingest Granules workflow', () => {
         expect(doesExist).toEqual(true);
       });
 
+      it('applyworkflow UpdateCmrAccessConstraints updates and publishes CMR metadata', async () => {
```

Comment:
This isn't really an issue directly with this PR, but a couple of default questions:

Nits: 
1) Is there a reason we need to run a full workflow here, or could we just test the deployed lambda component via a direct execution? 

I'm inferring from this that the goal is to test that CMR accepts the output from UpdateCmrAccessConstraints, so ... yes?       I'm adding this comment prior to reviewing the implementation code, so may remove before finishing the review
 
2. Aside from the boilerplate, and with the caveat that this isn't from this PR directly, is there a reason we have to pair this *in file* to this test?      Should we file a cleanup ticket and prioritize to make this not 1k lines long and testing multiple workflows, etc?

Reply:
1. To ensure that we fail testing if we make any changes to the code that end up failing CMR file validation (a major stumbling block when I started working on the integration side of things). We *need* to test actually submitting the updated file to CMR, and ensuring it passes validation.

2. I'll admit it's a bit of a convenience copout, but setting up an entirely separate test file that ingests a granule with a valid CMR file that is capable of being published was a lot more work for questionable benefit, in my opinion.